### PR TITLE
weekly-editorial-2022-06-30

### PR DESCRIPTION
**Pull request by inoxx03**
Draft for today's editorial.
Still needs a few article recommendations

### CODE REVIEW

**Comment by rpelisse** [comment]
@inoxx03 I think you forgot to finish your editorial! Can you finished that today? 
@LightGuard @oraNod can you merge (and release) when this is ready? I'll be offline soon... Thanks!

File: src/content/posts/weekly-2022-06-30.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
+= This Week in JBoss - June 30th 2022
+Stefan Sitani
+2022-06-30
+:tags: quarkus, java, kubernetes, openshift,
```
Comment (oraNod) [inline src/content/posts/weekly-2022-06-30.adoc:4]:
```suggestion
:tags: quarkus, java, kubernetes, openshift
```

File: src/content/posts/weekly-2022-06-30.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Hello and welcome to _This Week in JBoss_ June 30th edition! We are nearing the end of the second week of summer, and most of us are already looking forward to our vacation plans.
+And while for some the next 2 months will be a time to relax, slow down, and take things a little easy, progress and innovation never really stop!
+So for those of you interested in the latest and the greatest that your favorite project have to offer, here are this week's highlights from around the JBoss community.
```
Comment (oraNod) [inline src/content/posts/weekly-2022-06-30.adoc:8]:
```suggestion
So for those of you interested in the latest and the greatest that your favorite project has to offer, here are this week's highlights from around the JBoss community.
```

File: src/content/posts/weekly-2022-06-30.adoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+* link:https://quarkus.io/blog/quarkus-2-10-0-final-released/[Quarkus 2.10.0]
+
+
+== Articles & Blogs
+
+//TODO
+
+=== Videos
+
+This week there was plenty of fresh content to choose from, so pleas enjoy some of my top video picks:
```
Comment (oraNod) [inline src/content/posts/weekly-2022-06-30.adoc:34]:
```suggestion
This week there was plenty of fresh content to choose from, so please enjoy some of my top video picks:
```

**Review by oraNod — COMMENTED** [review]
Sorry I missed the "Draft" in the title. Here are some minors anyway. I can review again when you're out of draft mode.

**Comment by inoxx03** [comment]
@rpelisse Sure, I'll add the articles section this afternoon. Went over to deal with other TODOs and didn't get to come back to this.
Thank you @oraNod  for the suggested  changes.

**Comment by inoxx03** [comment]
Hi @oraNod @LightGuard Apologies for the delivery slip. I have added the article and blog recommendations and completed all the edits. Please feel free to sanity check the changes and merge this when you have a chance. Let me know if you find any any issues. Thank you.

File: src/content/posts/weekly-2022-06-30.adoc [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+=== Multi-cloud storage strategies for SaaS applications
+
+link:https://developers.redhat.com/articles/2022/06/23/multi-cloud-storage-strategies-saas-applications#[Multi-cloud storage strategies for SaaS applications] by Michael Hrivnak
+
+In the fourth entry to the ongoing link:https://developers.redhat.com/articles/2022/05/18/saas-architecture-checklist-kubernetes[_SaaS architecture checklist_] blog series Michael Hrivnak compares software-defined storage (SDS) technologies that help developers create optimized data storage solutions for use in multi-tenant cloud environments.
+The first part of Michael's article deals with the broader technical consideration of how SDS can help developers minimize platform-specific development work, followed by an overview of Red Hat's current could storage technology offerings. In the latter section, Michael provides some insight into combining these offerings to create solutions that provide customers with self-scaling, self-managed, low-latency data storage capabilities.
```
Comment (LightGuard) [inline src/content/posts/weekly-2022-06-30.adoc:42]: could -> cloud

**Review by LightGuard — CHANGES_REQUESTED** [review]

File: src/content/posts/weekly-2022-06-30.adoc (hunk 4 above)
Comment (inoxx03) [reply src/content/posts/weekly-2022-06-30.adoc]: Thank you @LightGuard, good catch, my brain and my spellchecker both missed this one:) Fixed!

**Pull request merged**